ckanext/switzerland/logic.py

```diff
@@ -7,7 +7,8 @@
 import ckan.plugins.toolkit as tk
 from ckan.lib.search.common import make_connection
 from ckanext.switzerland.helpers.request_utils import get_content_headers
-from ckanext.switzerland.helpers.frontend_helpers import get_dataset_count
+from ckanext.switzerland.helpers.frontend_helpers import get_dataset_count,\
```

Comment:
Could we just move those helpers? They are not frontend helpers anymore, technically they are also sort of logic functions, we just don't want them called from the outside, so what about logic_helpers?

I spent a lot of time detangeling the helper, to avoid cyclic errors, so I really care for putting them into the right file. What do you think?

Reply:
Yes, logic_helpers sounds good. I didn't think too much about this but you're right, they don't really fit in frontend_helpers. I'll move them.